php-practice/section7.php

```diff
@@ -76,3 +97,15 @@
  *
  * (It returns TRUE if $needle is a substring of $haystack; FALSE otherwise.)
  */
+
+function stringContains($haystack, $needle) {
+	$pos = strpos($haystack, $needle);
+
+		if ($pos == true) {
+			echo "TRUE";
+		} else {
+			echo "FALSE";
+		}
+}
```

Comment:
Yes I just looked that up because I couldn't get the function to return anything but false when I used ===. When you use a "loose equivalence operator" it won't distinguish between the substring's presence versus position.